README.md

```diff
@@ -14,3 +14,4 @@ Documentation related to podified controlplane.
 * [Operator controller manager pod labels](operator_controller_labels.md)
```

Comment:
.. and we can expand the title of the PR to cover CephFS other than rbd

Reply:
Can we please add manila in a follow up patch and not delay merging this one?